inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gather_nd.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/gather_nd.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> dPrecisions = {
+        InferenceEngine::Precision::FP32,
+        InferenceEngine::Precision::FP16,
+        InferenceEngine::Precision::I32,
+//        InferenceEngine::Precision::I64,
+        InferenceEngine::Precision::U8,
+        InferenceEngine::Precision::I8
+};
+const std::vector<InferenceEngine::Precision> iPrecisions = {
+        InferenceEngine::Precision::I32,
+        InferenceEngine::Precision::I64
+};
+
+const auto gatherNDArgsSubset1 = ::testing::Combine(
+        ::testing::Values(std::vector<size_t>({2, 2})),         // Data shape
+        ::testing::Values(std::vector<size_t>({2, 2})),         // Indices shape
+        ::testing::Values(std::vector<int>({0, 0, 1, 0})),   // Indices values
```

Comment:
fix code alignment here and below please

Reply:
Fixed